lib/hexpm_web/views/dashboard/audit_log_view.ex

```diff
@@ -0,0 +1,100 @@
+defmodule HexpmWeb.Dashboard.AuditLogView do
+  use HexpmWeb, :view
+
+  alias HexpmWeb.DashboardView
+
+  @doc """
+  Translate an audit_log to user readable descriptions
+  """
+  def humanize_audit_log_info(log = %{action: "docs.publish"}) do
+    "Publish documentation for #{log.params["package"]["name"]} (#{
```

Comment:
Do we need to handle the case when `params["package"]["name"]` is `nil`?

It won't crash now, but the humanized result would be weird: `Publish documentation for  ()`

Reply:
We should always have a name in this case.

Reply:
Then I'd suggest we keep the current simple logic and fix any issues as they occur, sounds good to you?